batcher/api/pom.xml

```diff
@@ -21,7 +21,7 @@
   <parent>
     <artifactId>dsbulk-batcher</artifactId>
     <groupId>com.datastax.oss</groupId>
-    <version>1.8.1-SNAPSHOT</version>
+    <version>1.8.0</version>
```

Comment:
Would you mind reverting this change and similar ones? I don't think they are required for the purpose of this PR.